backend/globaleaks/utils/utility.py

```diff
@@ -15,8 +15,11 @@
 import sys
 import traceback
 import uuid
+import platform
 from datetime import datetime, timedelta
 
+if platform.system() == 'Windows':
+    import ctypes
```

Comment:
Unused import ctypes pylint warning :) Course I can always silence pylint.

Reply:
ah, leave as it is if this is the case.

why pylint identify it as unused if you are using it in Windows?

Reply:
no, it would ident as unused on Linux ... I think?

It might be satisfied since the code is down lower even though it's in a if branch that never gets run on Linux. I do prefer not importing things if I don't need them though.

Reply:
fine for keeping the if! you are right! let's leave the import out :)